ports/stm32/mphalport.c

```diff
@@ -114,6 +114,12 @@ void mp_hal_pin_config(mp_hal_pin_obj_t pin_obj, uint32_t mode, uint32_t pull, u
     GPIO_TypeDef *gpio = pin_obj->gpio;
     uint32_t pin = pin_obj->pin;
     mp_hal_gpio_clock_enable(gpio);
+    if (mode == MP_HAL_PIN_MODE_ALT) {
```

Comment:
I think there needs to also be a check for `MP_HAL_PIN_MODE_ALT_OPEN_DRAIN` here.

Reply:
done